src/renderer/components/instance/ServersPage.ts

```diff
@@ -0,0 +1,13 @@
+import { customElement, html, LitElement, property, TemplateResult } from "lit-element";
```

Comment:
Codacy found an issue: ['property' is defined but never used.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=6041480)